### DISCUSSION TITLE
Multiline mappings?

### DISCUSSION THREAD

**Discussion by 097115** [andmarti1424/sc-im#706] [Ideas]
Just a thought: with a long mapping, for the sakes of readability, it would be great to have a opportunity to split it over several lines, like Vim can :)

May be something like this:

        # insert date in the first column
        nnoremap "id" "<HOME>
                     \ =@ston(@ext(\"date +'%s' #\", 0))<CR>
                     \ :valueize<CR>
                     \ :format \"d%Y-%m-%d\"<CR>"

We can't do anything similar at the moment, right?

**Comment by andmarti1424** [comment]
I am afraid no. Move to discussion with other feature requests..
